Fix typo for "捕捉"

botamochi0x12 opened the pull request:
Replaced "補足" to "捕捉" when the original word is either "catch" or "capture".

原文で「catch」か「capture」が使われている場面で「補足」を「捕捉」に置き換えました。

ちなみに数カ所ほど行末の半角スペースが切り取られて(`trailing-white-space`)しまっていますので、
気になさるならばリジェクトしてください。

botamochi0x12 committed:
Fix typo for "捕捉"

Replaced "補足" to "捕捉" when the original word is either "catch" or "capture".

原文で「catch」か「capture」が使われている場面で「補足」を「捕捉」に置き換えました。

dalance commented:
行末の空白削除ですが、以下の通り原文リポジトリとの一致チェックに引っかかってしまうので
もとに戻していただけると助かります。

https://app.circleci.com/pipelines/github/rust-lang-ja/rust-by-example-ja/170/workflows/462b5ddd-c75f-4b2f-b2dc-9f0c4c6469fb/jobs/349/parallel-runs/0/steps/0-108

botamochi0x12 committed:
Restore trailing spaces

The trailing spaces are necessary for identidication 
with the original repository on English.

botamochi0x12 commented:
行末の半角スペースを元に戻しました。
同一性チェックも通るようになったみたいですね。

dalance commented:
ありがとうございます。マージします。
